apps/frontend/src/assets/css/variables.css

```diff
@@ -17,8 +18,8 @@
 	--physical-end: #fff8b5;
 	--work-start: #69ff35;
 	--work-end: #c3ff19;
-	--friend-start: #7f21ce;
-	--friend-end: #cb00ff;
+	--friends-start: #7f21ce;
```

Comment:
This might break gradients for others

Reply:
I agree. I used these colors for sliders. With updated name, then we will need to update these colors across the app.